### ISSUE TITLE
Issues following documentation on report Sections

### ISSUE THREAD

**Issue by jacksonjw12** [dolanmiu/docx#864]
Hello,
I have been using this library for a few days, and I have experienced a couple issues following the documentation. The most notable is that I cannot specify a Section property type like in https://docx.js.org/#/usage/sections because the SectionType class is not exported from the package. I tried to bypass this by doing something like the following, however the result is each section is placed on a new page, not continuously. 

```
this.doc.addSection({
        properties: {type:'continuous'},
        children: [
          new Paragraph({
            children: [
              new TextRun({
                text: "Test",
                bold: true,
              }),
            ],
          }),
          new Paragraph({children:[
            new TextRun({
              text: `123`,
              bold: true,
            }),
          ]})
        ],
      });
```

The other issue is that even following the basic usage instructions documentation results in a blank document. I took the following code directly from the basic usage instructions https://github.com/dolanmiu/docx/tree/master/docs and it still results in a blank document.
```
import { Document, Packer, Paragraph, TextRun, Media } from "docx";
import * as fs from "fs";
let name = "test.docx"

const doc = new Document({
  sections: [{
      properties: {},
      children: [
          new Paragraph({
              children: [
                  new TextRun("Hello World"),
                  new TextRun({
                      text: "Foo Bar",
                      bold: true,
                  }),
                  new TextRun({
                      text: "\tGithub is the best",
                      bold: true,
                  }),
              ],
          }),
      ],
  }]
});

Packer.toBuffer(doc).then((buffer)=>{
  fs.writeFileSync(name, buffer);
  
})
```

Could you please give an example of a working method of adding continuous sections to a document? All examples in the repo seem to be directly importing the SectionType class from within the source files itself. Also if you could point out why the example from the docs is generating a blank document that would be helpful.

**Comment by jacksonjw12** [comment]
Looks like this was also an issue seen in https://github.com/dolanmiu/docx/issues/790 . Is there any way to make report sections continuous in the current release? I guess the workaround would be to use one report section for the whole document.

**Comment by jacksonjw12** [comment]
Looks like this is fixed in 6.0.2
